src/main/java/frc/robot/vision/multivisionsources/MultiVisionSources.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.vision.multivisionsources;
+
+import frc.robot.subsystems.GBSubsystem;
+import frc.robot.vision.VisionConstants;
+import frc.robot.vision.rawdata.VisionData;
+import frc.robot.vision.sources.VisionSource;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+
+public class MultiVisionSources<ReturnType extends VisionData> extends GBSubsystem {
```

Comment:
Why is this a subsystem?
Also, doesn't a multi-vision source should behave like a source?

Reply:
MultiVisionSources does not behave like a source since it doesn't return an optional (it filters empty optionals. An isEmpty trait would be cool) and doesn't need manual updating. It's a subsytem for logging periodically.